index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ideefase PDF Generator</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.4.0/jspdf.umd.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="container">
+    <form id="ideefaseForm" class="space-y-6" onsubmit="return validateForm();">
+    <!-- Header -->
+    <div class="header">
+        <header class="text-center">
+            <h1 class="title">Ideefase PDF Generator</h1>
+            <p class="subtitle">Genereer een overzichtelijk ideefase PDF</p>
+        </header>
+    </div>
+
+        <!-- Form Section -->
+    <div class="form-section">
+
+            <!-- Project Type -->
+            <div>
+                <label class="form-label">Project Type</label>
+                <p class="label-subtitle">Welke eindopdracht ga je maken?</p>
+                <div class="radio-group">
+                    <label class="radio-label">
+                        <input type="radio" name="projectType" value="Backend" checked>
```

Comment:
`<input />` is een self-closing element maar wordt niet gesloten (geldt voor allemaal!)